lib/fluent/command/plugin_generator.rb

```diff
@@ -0,0 +1,229 @@
+#
+# Fluentd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require "optparse"
+require "pathname"
+require "fileutils"
+require "erb"
+
+class FluentPluginGenerator
+  attr_reader :type, :name
+  attr_reader :license
```

Comment:
It's ok to fix the license as `Apache-2.0`.
If this feature supports GPLv2 or GPLv3 or any others, it means that Fluentd source code tree have the license files/templates of these licenses, or code to download these license files. It sounds very bad situation.